releasenotes/notes/remove-tracer-deprecations-c0abaecda9c24b79.yaml

```
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    Remove deprecated attribute ``ddtrace.Tracer.debug_logging``. Set the logging level for the ``ddtrace`` logger instead::
+
+      import logging
+      log = logging.getLogger("ddtrace")
```

```suggestion
      log = logging.getLogger("ddtrace.tracer")
```

probably want to confirm this works too